packages/react/src/components/Modal/Modal-story.js

```diff
@@ -68,6 +68,32 @@ const props = () => ({
   onSecondarySubmit: action('onSecondarySubmit'),
 });
 
+const titleOnlyProps = () => ({
+  className: 'some-class',
+  open: boolean('Open (open)', true),
+  passiveModal: true,
+  danger: boolean('Danger mode (danger)', false),
+  modalHeading: text(
+    'Modal heading (modalHeading)',
+    `
+    Passive modal title as the message. Should be direct and 3 lines or less.
+  `.trim()
+  ),
+  modalAriaLabel: text(
+    'ARIA label, used only if modalLabel not provided (modalAriaLabel)',
+    'A label to be read by screen readers on the modal root node'
+  ),
+  size: select('Size (size)', sizes, 'sm'),
+  iconDescription: text(
```

Comment:
Does this need to be a knob?

Reply:
Yes so people can test change in the prop.